assembly/pom.xml

```diff
@@ -143,7 +144,6 @@
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-shade-plugin</artifactId>
```

Comment:
it is now managed using plugin management in the top-level module (all versions are)